src/main/java/com/redhat/cloud/policies/app/model/Policy.java

```diff
@@ -255,14 +255,11 @@ public void delete(Policy policy) {
     }
 
     public void populateFrom(Policy policy) {
-        this.id = policy.id;
```

Comment:
`this` comes from the DB and already has an `id` value. It doesn't make sense to change it here.